packages/block-editor/src/components/block-list/style.scss

```diff
@@ -74,7 +74,7 @@
 	&.is-navigate-mode .block-editor-block-list__block.is-selected::after,
 	& .is-block-moving-mode.block-editor-block-list__block.has-child-selected {
 		box-shadow: 0 0 0 var(--wp-admin-border-width-focus) var(--wp-admin-theme-color);
-		outline: var(--wp-admin-border-width-focus) solid transparent;
+		outline: 2px solid transparent;
```

Comment:
For clarity: the transparent outline exists for the purpose of enabling a focus style in Windows using High Contrast Mode. In that mode, all box shadows are removed, and all outlines/borders are made a highly visible yellow color (on black background). 

In other words, by serving a _transparent_ outline style, it's invisible for people not using high contrast mode, just like how the box shadow is invisible for people who do.